modules/dynamic-plugins/con-rhdh-plugins.adoc

```diff
@@ -0,0 +1,64 @@
+
+A key feature of {product-very-short} is the ability to import your own custom built or third-party plugins or to create new features using dynamic plugins.
+
+Dynamic plugins boost modularity and scalability by enabling more flexible and efficient functionality loading, significantly enhancing the developer experience and customization of your {product-very-short} instance.
+
+== Dynamic plugins
+{product-very-short} dynamic plugins offer several benefits, particularly in environments where flexibility, scalability, and customization are key. Here are some of the benefits that using dynamic plugins in {product-very-short} provide:
+
+Modularity and extensibility::
+Dynamic plugins allow teams to add or modify features without altering the core {product-very-short} application. This modular approach makes it easier to extend functionality as needs evolve.
+
+Customization::
+Different teams or projects might have unique requirements. Dynamic plugins enable users to tailor {product-very-short} to fit specific workflows and use cases, enhancing the overall user experience.
+
+Reduced maintenance and update overhead:: 
+With dynamic plugins, updates or new features can be deployed independently of the main {product-very-short} codebase. This reduces the risks and efforts associated with maintaining and updating the platform.
+
+Faster iteration:: 
+Developers can create and test new features more rapidly as plugins. This encourages experimentation and innovation, allowing teams to quickly iterate based on feedback.
+
+Improved collaboration::
+Organizations can share plugins across teams or even externally. This sharing can foster collaboration and reduce duplication of effort, as well as help establish best practices across an organization.
+
+Scalability:: 
+As organizations grow, their needs become more complex. Dynamic plugins enable {product-very-short} to scale alongside these needs, accommodating an increasing number of users and services.
+
+Ecosystem growth:: 
+Encouraging the development of plugins can lead to a vibrant ecosystem around {product-very-short}. This community can contribute plugins that address a wide range of needs, further enriching the platform.
+
+Security and compliance:: 
+Plugins can be developed with specific security and compliance requirements in mind, ensuring that {product-very-short} installations meet the necessary standards without compromising the core application.
+
+Overall, the use of dynamic plugins in {product-very-short} promotes a flexible, adaptable, and sustainable approach to managing and scaling development infrastructure.
+
+== Comparing dynamic plugins to static plugins
+Static plugins are built into the core of the {product-very-short} application. Installing or updating a static plugin requires a restart of the application after installing the plugin.
```

Comment:
> Installing or updating a static plugin requires a restart of the application after installing the plugin.

This sentence is true for dynamic plugins but not statically integrated plugins.  Installing or updating a statically integrated plugin requires updating the Developer Hub application source code and rebuilding the application and container image.